### PR TITLE
Add support for taxon slug changes (closes #38)

### LINKED ISSUE #38: [ENHANCEMENT] Add a listener on taxon slug change to create automatic redirection
Same as the one for products, we should now create it for Taxon.

### PR DESCRIPTION
**Pull request by stefandoorn**
Fixes #38

### CODE REVIEW

**Comment by loevgaard** [comment]
Thank you, Stefan :) We will look into this PR as soon as we can :)

**Comment by loevgaard** [comment]
Hi Stefan. Can you rebase now? We've just merged a PR that fixes the build

File: README.md [hunk 1]
```diff
@@ -109,11 +109,12 @@ Example: Having a slug like `/products/a`, renaming it to `/products/b` then ren
 
 ### Points of improvements 
 
-The same mechanism could be made for taxon slugs, and obviously for any other resource that has a slug. Feel free to submit a PR.
+The same mechanism could be made other resources that has a slug. Feel free to submit a PR.
```
Comment (loevgaard) [inline README.md:112]: For now we can just remove this whole section because there are no other core resources that has slugs I guess?

Comment (stefandoorn) [reply README.md:112]: True, I was thinking to look into the BitBag CMS, but that probably should be a stand alone repository.

**Pull request merged**